docs/contracts/02_docs/src/modules/PRICE/OlympusPrice.v2.sol/contract.OlympusPricev2.md

```
@@ -0,0 +1,1240 @@
+
+| Name     | Type      | Description              |
+| -------- | --------- | ------------------------ |
+| `asset_` | `address` | The address of the asset |
+
+**Returns**
+
+| Name     | Type      | Description                                                 |
+| -------- | --------- | ----------------------------------------------------------- |
+| `<none>` | `uint256` | price The USD price of the asset in the scale of `decimals` |
+
+### getPrice
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)